.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "printWidth": 119
```

Comment:
After reading https://prettier.io/docs/en/options.html#print-width do you know how different it'd be if we'd just have the length set with eslint?

Reply:
From my experience that would cause the editor to format the code using the default 80 chars per line when using the automatic formatting, it takes `.prettierrc` file as source of truth to do it.